include/Song.h

```diff
@@ -238,7 +238,7 @@ class LMMS_EXPORT Song : public TrackContainer
 			m_loopRenderCount = count;
 		m_loopRenderRemaining = m_loopRenderCount;
 	}
-    
+		
```

Comment:
No whitespace needed in the whole line (`git diff` marks all trailing whitespace red, which is why we usually keep changes clean of them).